ssz-rs/src/list.rs

```diff
@@ -92,12 +92,7 @@ where
     T: SimpleSerialize + Default,
 {
     fn default() -> Self {
-        let data = vec![];
-        match Self::try_from(data) {
-            Ok(result) => result,
-            // TODO: not ideal to panic here...
-            Err((_, err)) => panic!("{err}"),
-        }
+        Self { data: vec![], cache: MerkleCache::with_leaves(0) }
```

Comment:
this works! I can pull into #55

my reasoning there was to keep the symmetry with the one "entry point" for all `List` types in the `try_from` impl but its fine to inline this here